cypress.config.js

```diff
@@ -0,0 +1,9 @@
+import { defineConfig } from "cypress";
+
+export default defineConfig({
+  e2e: {
```

Comment:
Use this config to set the baseURL. https://docs.cypress.io/guides/references/best-practices#Setting-a-Global-baseUrl